install_config/oab_broker_configuration.adoc

```diff
@@ -553,6 +553,78 @@ is also required to configure the broker to use the Partner Registry URL:
     registry.connect.redhat.com
 ----
 
+[[oab-config-helm-chart-registry]]
+=== Helm Chart Registry
+
+Using the `helm` type allows you to consume Helm Charts from a Helm Chart Repository.
+
+[source,yaml]
+----
+registry:
+  - name: stable
+    type: helm
+    url: "https://kubernetes-charts.storage.googleapis.com"
+    runner: "docker.io/automationbroker/helm-runner:latest"
+    white_list:
+      - ".*"
+----
+
+*Note:* A heavy percentage of helm charts from the stable repository are not well suited for OpenShift.
```

Comment:
1. Can you replace `*Note:* A heavy...` with `NOTE: A heavy...` so that it renders as a note?
2. Can you replace `OpenShift` with `{product-title}`?